packages/devtools/lib/src/profiler/cpu_profiler.dart

```diff
@@ -190,6 +191,8 @@ class CpuProfilerTabNav {
 
   final CpuProfilerTabOrder tabOrder;
 
+  final TreeTableToolbar treeTableToolbar = TreeTableToolbar<CpuStackFrame>();
```

Comment:
You are throwing away static type information here.

The idiomatic way to write this is:

`final treeTableToolbar = TreeTableToolbar<CpuStackFrame>();`

Also acceptable is the unnecessarily verbose:

`final TreeTableToolbar<CpuStackFrame> treeTableToolbar = TreeTableToolbar<CpuStackFrame>();`

or the slightly less verbose:

`final TreeTableToolbar<CpuStackFrame> treeTableToolbar = TreeTableToolbar();`


When you have an explicit type on the left, inference can flow the generic type _to_ the right, but not _from_ the right. As written you have a _runtime_ `TreeTableToolbar<CpuStackFrame>` but a _static_ `TreeTableToolback<dynamic>`